libraries/AP_L1_Control/AP_L1_Control.h

```diff
@@ -128,4 +129,7 @@ class AP_L1_Control : public AP_Navigation {
     bool _reverse = false;
     float get_yaw() const;
     int32_t get_yaw_sensor() const;
+
+    // allow loiter radius scaling to be disabled (in path guidance).
+    bool _disable_loiter_radius_scaling = false;
```

Comment:
doesn't need the = false (wastes flash)

Reply:
Removed in 58a7602105ac8661bc2880cb18fee0bb0f94583e